engine: test config colors of "@log.cli" not passing

When:
```
cd packages/engine/env/ubuntu_trusty
docker-compose up --abort-on-container-exit
```

Error:
```
1) actions.log.cli
nodejs_1  |        handled event
nodejs_1  |          option colors (0.local):
nodejs_1  | 
nodejs_1  |       AssertionError: expected Array [
nodejs_1  |   'localhost   a   -\n',
nodejs_1  |   'localhost   b   ✔\n',
nodejs_1  |   'localhost   c   ✘\n'
nodejs_1  | ] to equal Array [
nodejs_1  |   '\u001b[36m\u001b[2mlocalhost   a   -\u001b[22m\u001b[39m\n',
nodejs_1  |   '\u001b[32mlocalhost   b   ✔\u001b[39m\n',
nodejs_1  |   '\u001b[31mlocalhost   c   ✘\u001b[39m\n'
nodejs_1  | ]
```

--- a/packages/engine/env/centos7/Dockerfile
+++ b/packages/engine/env/centos7/Dockerfile
@@ -29,6 +29,7 @@ RUN yum clean all
 ADD ./run.sh /run.sh
 RUN mkdir -p /nikita
 WORKDIR /nikita/packages/engine
+# Enable color output in TTY
 ENV TERM xterm
 
 #CMD ["node_modules/.bin/mocha", "test/api/"]

--- a/packages/engine/env/ubuntu_trusty/Dockerfile
+++ b/packages/engine/env/ubuntu_trusty/Dockerfile
@@ -18,5 +18,7 @@ RUN apt-get install -y openssh-server \
 ADD ./run.sh /run.sh
 RUN mkdir -p /nikita
 WORKDIR /nikita/packages/engine
+# Enable color output in TTY
+ENV TERM xterm
 
 ENTRYPOINT ["/run.sh"]

